tsl/src/fdw/option.c

```diff
@@ -145,6 +146,18 @@ option_validate(List *options_list, Oid catalog)
 			/* check and store list, warn about non existing tables */
 			(void) option_extract_join_ref_table_list(defGetString(def));
 		}
+		else if (strcmp(def->defname, "copy_rows_per_message") == 0)
+		{
+			int copy_rows_per_message;
+
+			copy_rows_per_message = strtol(defGetString(def), NULL, 10);
```

Comment:
nit: it could also return `LONG_MAX` on overflow and set errno as `ERANGE`, maybe it should be checked as well

Reply:
If LONG_MAX is returned, I believe that just means we'd use the max number of rows per message even if the user input a bizarrely large number. Not sure we need to notify the user for that odd case.

I also just followed the same code as for other parameters, like fetch_size, which originally came from postgres_fdw.